File: 1791-richest-customer-wealth/1791-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max_wealth=0;
        for(int i=0;i<accounts.length;i++){
            int wealth=0;
            for(int j=0;j<accounts[i].length;j++){
                wealth+=accounts[i][j];
            }
            if (wealth>=max_wealth){
                max_wealth=wealth;
            }
        }
        return max_wealth;

    }
}

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        res=0
        for i in range(len(accounts)):
            s=0
            for j in range(len(accounts[i])):
                s+=accounts[i][j]
            if s>=res:
                res=s
        return res

        